.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
```

Comment:
I think you have to add the following as well:
- react 
- node

Reply:
I think `react` is recognized as a plugin rather than an env variable. throws me an error when I try to add `react: true` to the `env` block.